script/tool/test/build_examples_command_test.dart

```diff
@@ -379,7 +379,6 @@ void main() {
         ]),
       );
 
-      print(processRunner.recordedCalls);
```

Comment:
Thanks, I thought I'd caught all of these during all the refactoring/cleanup work. Apparently it spread to more places than I'd noticed.

Reply:
there's still some prints somewhere in the tests but i couldn't track them down in the 10 seconds i spent trying. It says "Determine diff with base sha:" sometimes, and "null" once.

Reply:
> "Determine diff with base sha:" sometimes

I bet that's from a test of the helper directly, not going through a command runner that captures output. That helper really shouldn't log anyway, I just haven't gotten around to fixing it.